### ISSUE TITLE
Unable to see ripple or underline on Tabs. Example codepen doesn't even work.

### ISSUE THREAD

**Issue by mr-e-** [react-mdl/react-mdl#415]
https://tleunen.github.io/react-mdl/components/tabs/

On this link, while it works on this page, if you click the first example codepen, it does not work. There is no underline and no ripple. This is the same effect I got when I tried to place into my code.

**Comment by hmaciasc** [comment]
I can confirm not even adding ripple fixes it

**Comment by StrikeForceZero** [comment]
a dirty fix

```js
private componentDidMount() {
   window.componentHandler.upgradeAllRegistered();
}
```

**Comment by DirtyHairy** [comment]
This is a duplicate of #394 that got fixed in 1.8.0 .
